packages/components/filters/src/filter-menu/footer/footer.tsx

```diff
@@ -1,5 +1,70 @@
-function Footer() {
-  return <div>footer</div>;
-}
+import FlatButton from '@commercetools-uikit/flat-button';
+import messages from './messages';
+import { CloseBoldIcon } from '@commercetools-uikit/icons';
+import { ReactNode } from 'react';
+import { css } from '@emotion/react';
+import { designTokens } from '@commercetools-uikit/design-system';
+import { useIntl } from 'react-intl';
 
+type TFooterProps = {
+  /**
+   * controls whether `apply` button in Menu Body Footer is displayed
+   */
+  renderApplyButton?: () => ReactNode;
+  /**
+   * controls whether a `clear all` button in Menu Body Footer is displayed
+   */
+  onClearAllRequest?: () => void;
+};
+
+const mainContainerStyles = css`
+  height: auto;
+  width: 100%;
+   };
+`;
+const footerContainerStyles = css`
+  /* horizontal divider */
+  height: ${designTokens.borderWidth1};
+  background-color: ${designTokens.colorNeutral90};
+  margin-top: ${designTokens.spacingS};
+
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  align-items: center;
+
+  & > div:first-of-type {
+    margin-top: ${designTokens.spacingS};
+  }
+
+  & > div:last-of-type {
+    margin-top: ${designTokens.spacingS};
+    display: flex;
+    justify-content: flex-end;
+  }
```

Comment:
- the t-shirt size named spacing-tokens were deprecated at some point in favor of numeral ones (`spacing20` is the equivalent of `spacingS`) 
- you can use the `padding-top` property on the parent `div` to apply the safespace instead of setting the `margin-top` property two times on the 2 child-div's.
- elements in ui-kit which need styling are usually targeted directly and not via `&` selectors (by avoiding those selectors all css-rules generated have the same specificity).

Reply:
@FilPob  one more thing I noticed is that the Clear-Button has a different computed height than the Apply button. In case there is no apply button, this will cause the footer to have a different (lower) height than with two buttons. It's probably worth to apply a fixed height to the footer and position the buttons within.

Reply:
@misama-ct no that's okay if the footer is smaller, if there is only the clear all button